common/types/admin.go

```diff
@@ -394,8 +394,9 @@ type IsolationGroupPartition struct {
 // Indicating that task processing isn't to occur within this isolationGroup anymore, but all others are ok.
 type IsolationGroupConfiguration map[string]IsolationGroupPartition
 
+// ToPartitionList Renders the isolation group to the less complicated and confusing simple list of isolation groups
 func (i IsolationGroupConfiguration) ToPartitionList() []IsolationGroupPartition {
-	var out []IsolationGroupPartition
+	out := []IsolationGroupPartition{}
```

Comment:
this is intentional, there was a type bug which was revealed during drain removal when this was nil